### DISCUSSION TITLE
FloodWaitError Occured will not auto reconnect ?

### DISCUSSION THREAD

**Discussion by warmkenley** [aahnik/tgcf#7] [Q&A] [answered]
Hello, After running a new group forwarding. Just notice below error and get disconnect, 

```
**2021-01-08 01:26:11,437 - root - INFO - forwarding message with id = 6456

A wait of 1246 seconds is required (caused by SendMediaRequest)

Run the script again after some time.                         FloodWaitError Occured
2021-01-08 01:26:11,615 - telethon.network.mtprotosender - INFO - Disconnecting from xxx.xxx.xxx.xx:443/TcpFull...
2021-01-08 01:26:11,615 - telethon.network.mtprotosender - INFO - Disconnection from xxx.xxx.xxx.xx:443/TcpFull complete!**
```
but after waiting 1246seconds it will not auto reconnect itself, it there any solution?
as I am not always with my computer that can connect the VPS, will be great if it can reconnect with some time itself

anyway I will make a review after I done all message forwarded later and will see if this occur with new message

thank you so much for sharing this chat forward script!!

**Comment by aahnik** [comment]
This script is meant to run and finish. So when a flood error occurs, the program intentionally quits instead of waiting such many seconds.

This behaviour can be changed easily. Look at the place of the code where I catched the flood wait exception. Instead of quitting, wait for the required seconds. You need some basic python knowledge and have to read the Telethon Documentation.


------

By the way, I have another seperate script that **runs constantly ( live sync, real time forwarding )** . And can be deployed to Heroku, by a click of button. That is currently closed source for some reasons, and if you want that contact me on [Telegram @AahnikDaw](https://telegram.me/aahnikdaw)

**Comment by aahnik** [comment, marked as answer]
**This problem has been fixed.** Clone or download the latest version. It will automatically wait for required time, on hitting FWE, and then continue the work.

**Reply by warmkenley** [reply]
Hi @aahnik , Sorry for late reply, on 8 Jan same day I tried to reconnect manually, but soon my tg account get banned after forwarding like around 12000message from same group, therefore i think it will be good if I can select which type of message I need auto forward.

Anyway it is very lucky I used a new created account to run the program so no issue to my main account, i believe you did not faced this issue but it will be good if you will add some warning message to other user about this to be careful

**Reply by aahnik** [reply]
in future, I will try to support the feature you requested
